test/integration/user_can_checkout_order_test.rb

```diff
@@ -26,4 +28,33 @@ class UserCanCheckoutOrder < ActionDispatch::IntegrationTest
     assert page.has_content? order.total
     assert_equal order.status, "ordered"
   end
+
+  test "unregistered user redirects back to duffel after creating an account" do
+    user = create(:user)
+    create(:item)
+
+    visit items_path
+    click_on "Add to Duffel"
+    click_on "Add to Duffel"
+    visit duffel_path
+    click_on "Checkout"
+
+    assert_equal login_path, current_path
+
+    within '.create-acct' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.